local/tests/runit.dotproduct.R

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
We don't need this file.  Check `git blame` on the others -- this was for the predecessor to `tinytest`.